source/unit/Constraints.ooc

```diff
@@ -131,6 +139,10 @@ EqualModifier: class extends Modifier {
 		f := func (value, c: Object) -> Bool { value as Cell<Double> get() == c as Cell<Double> get() }
 		CompareWithinConstraint new(this, Cell<Double> new(correct), f)
 	}
+	to: func ~ldouble (correct: LDouble) -> CompareWithinConstraint {
+		f := func (value, c: Object) -> Bool { value as Cell<LDouble> get() == c as Cell<LDouble> get() }
+		CompareWithinConstraint new(this, Cell<LDouble> new(correct), f)
+	}
```

Comment:
Exactly. This would be something like
```C++
template <typename T>
CompareWithConstraint to(const T correct) {
    auto f = [&](const T value, Object& c){ return (Cell&)(value) . get<T>() == (Cell&)(c) . get<T>(); }
   return CompareWithinConstraint new(this, new Cell<T>(correct), f);
}
```